.gitignore

```diff
@@ -23,3 +23,9 @@ yarn-error.log*
 *.sw*
 
 .directory
+
+/functions/lib
+
+// config for local functions
+audio-seva-team*.json
```

Comment:
I don't think that adding a specific project name into git config is a good idea, What kind of a file is it?

Reply:
The file is from https://firebase.google.com/docs/functions/local-emulator#set_up_admin_credentials_optional

Reply:
Do you mean that `audio-seva-team*.json` is a service account key file that you save into the project directory and thus add to `gitignore`? In this case you should save the file somewhere outside the project folder. The idea is that the source code (including `gitignore` file) should not include any artifact of the test firebase project.

Reply:
So please remove this.